### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by zonuexe**
EditorConfig helps to resolve discord by indentation rule.
http://editorconfig.org/

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[*.php]
+indent_style = tab
+indent_size = 8
```
Comment (zonuexe) [inline .editorconfig]: I thought that indent size is 8 when I look at GitHub, but we can change this number according to your preference.

Comment (zonuexe) [inline .editorconfig]: This setting affects the indentation width of tabs in GitHub.
https://stackoverflow.com/questions/8833953/how-to-change-tab-size-on-github

Comment (manicki) [inline .editorconfig]: I am not sure if there is any preference for this. Do we need to set it at all or could we let all tools display tabs as wide as they "normally" do?

Comment (thiemowmde) [inline .editorconfig]: https://www.mediawiki.org/wiki/Manual:Coding_conventions#Tab_size suggests 4 spaces. I ended merging this patch and editing it myself, see 8ae8c2af7a323d4ee094bd1db14f2417720c3fde. Feel free to upload another patch if you disagree with what I did. The easiest solution is to remove controversial lines.

**Pull request merged**